cmd/minishift/cmd/start.go

```diff
@@ -221,6 +222,7 @@ func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
 		DiskSize:         calculateDiskSizeInMB(viper.GetString(startFlags.DiskSize.Name)),
 		VMDriver:         viper.GetString(startFlags.VmDriver.Name),
 		DockerEnv:        dockerEnv,
+		EngineOpt:        engineOpt,
```

Comment:
"Engine" seems a bit too generic to me. I think either "DockerOpt" or "DockerEngineOpt" to make it clear to what this really applies.